pkg/osquery/table/mdfind_darwin.go

```diff
@@ -12,7 +12,7 @@ import (
 )
 
 func mdfind(args ...string) ([]string, error) {
-	ctx, cancel := context.WithTimeout(context.Background(), 3*time.Second)
+	ctx, cancel := context.WithTimeout(context.Background(), 30*time.Second)
```

Comment:
we see a lot of timeouts for mdfind, will there significant perf impacts to devices by 10x-ing this timeout? Does mdfind use significant CPU? ( it seems like this is a potential CPU hotspot depending on the query)

Reply:
No clue. I just upped the numbers.

Reply:
Fair enough. I don't think we have significant safety nets for other performance sensitive areas, so this is probably in line with current art.